extensions/adobe/experience/adcloud/search/profile-adcloud.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/search/profile-adcloud",
```

Comment:
Proifile is loaded jargon. It shouldn't be part of your $id especially since it doesn't have anything to do with the existing profile class.

Reply:
Removed ad-cloud defined class